contrib/endpoints/src/api_manager/request_handler.cc

```diff
@@ -29,6 +29,17 @@ using google::devtools::cloudtrace::v1::Traces;
 namespace google {
 namespace api_manager {
 
+RequestHandler::RequestHandler(ApiManagerImpl* api_manager,
+                               std::shared_ptr<CheckWorkflow> check_workflow,
+                               std::unique_ptr<Request> request_data)
+    : api_manager_(api_manager),
+      check_workflow_(check_workflow),
+      request_data_(std::move(request_data)) {
+  if (api_manager->IsConfigLoadingSucceeded()) {
```

Comment:
Hi @mangchiandjjoe 

Here:  https://github.com/cloudendpoints/esp/blob/master/src/nginx/grpc.cc#L138

we may have to check api_manager loading status,   If it is in progress,  delay it.

Then, create trancode_factory here